### PR TITLE
feat(images): add loader for several types of images

### LINKED ISSUE #39: Improve way how it handles the SVG files
Currently, there is only one way to import SVGs, which converts to a React Component. We've faced some issues while trying to import as path, like a regular file. So the idea here is to improve how we handle and make sure that all kind of assets is working fine:

For SVGS files: https://www.npmjs.com/package/@svgr/webpack
```jsx
import starUrl, { ReactComponent as Star } from './star.svg'
const App = () => (
  <div>
    <img src={starUrl} alt="star" />
    <Star />
  </div>
)
``` 

For images:
https://www.npmjs.com/package/next-images#usage

### PR DESCRIPTION
**Pull request by sastromo**
Closes #39 

[ISSUE: Improve way how it handles the SVG files #39](https://github.com/Significa/significa-start/issues/39)

Screenshot 2020-07-30 at 11 33 48

### CODE REVIEW

**Comment by sastromo** [comment]
> Awesome, great job

thank you!

File: src/templates/next/next.config.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: [
-        {
-          loader: '@svgr/webpack',
-          options: {
-            replaceAttrValues: { '#000': 'currentColor' },
+    config.module.rules.push(
+      {
+        test: /\.(png|jpe?g|gif)$/i,
```
Comment (pbrandone) [inline src/templates/next/next.config.js:6]: do we need this?
didn't know that next didn't handle these files automatically

File: src/templates/next/next.config.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-        },
-      ],
-    })
+        ],
+      },
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: '@svgr/webpack',
```
Comment (pbrandone) [inline src/templates/next/next.config.js:17]: please configure svgr to `replaceAttrValues: { '#000': 'currentColor' },`

File: src/templates/next/next.config.js (hunk 1 above)
Comment (danilowoz) [reply src/templates/next/next.config.js:6]: NextJS doesn't handle the image import, in order to do it, it needs the nextjs-image plugin to work properly, which basically does the same thing we have here.

File: src/templates/next/next.config.js (hunk 2 above)
Comment (sastromo) [reply src/templates/next/next.config.js:17]: done 👍

File: src/templates/next/next.config.js (hunk 1 above)
Comment (sastromo) [reply src/templates/next/next.config.js:6]: can I close this issue?

**Pull request merged**